frontend/sass/pages/create-team.scss

```diff
@@ -1,8 +1,26 @@
 .choose-team-form {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
   .choose-team-title {
     margin-bottom: 50px;
   }
+
+  .errorlist {
+    background-color: $orange;
+    color: #ffffff;
```

Comment:
Let's keep the consistency and put all color codes as variables in the variables file!